doc/manual/manual.tex

```diff
@@ -2268,10 +2268,152 @@ \subsection{Particles}
+
+\begin{verbatim}
+docker container prune
+\end{verbatim}
+or
+\begin{verbatim}
+docker container rm CONTAINER_NAME
+\end{verbatim}
+\end{enumerate}
+
+You are all set. Repeat steps 1-4 of this process as necessary when updating your model parameters. 
+
+\subsubsection{Developing \aspect{} within a container}
+
+The above given workflow does not include advice on how to modify \aspect{} inside the container. We recommend a slightly different workflow for advanced users that want to modify parts of \aspect{}. The \aspect{} docker container itself is build on top of a deal.II container that contains all dependencies for compiling \aspect{}. Therefore it is possible to run the deal.II container, mount an \aspect{} source directory from your host system and compile it inside of the container. An example workflow could look as following (assuming you navigated in a terminal into the modified \aspect{} source folder):
+
+\begin{verbatim}
+docker pull dealii/dealii:v8.5.pre.4-gcc-mpi-fulldepsmanual-debugrelease
+docker run -it -v "$(pwd):/home/dealii/aspect:ro" \
+  dealii/dealii:v8.5.pre.4-gcc-mpi-fulldepsmanual-debugrelease bash
+\end{verbatim}
+
+Inside of the container you now find a read-only \aspect{} directory that contains your modified source code. You can compile and run a model inside the container, e.g. in the following way:
+
+\begin{verbatim}
+mkdir aspect-build
+cd aspect-build
+cmake -DCMAKE_BUILD_TYPE=Debug -DDEAL_II_DIR=$HOME/deal.II-install $HOME/aspect
+./aspect $HOME/aspect/cookbooks/shell_simple_2d.prm
+\end{verbatim}
+
+To avoid repeated recompilations of the \aspect{} source folder we recommend to reuse the so prepared container instead of starting new containers based on the deal.II image. This can be achieved by the following commands outside of the container:
+
+\begin{verbatim}
+docker ps -a # Find the name of the running / recently closed container in the output
+docker restart CONTAINER_NAME
+docker attach CONTAINER_NAME
+\end{verbatim}
+
+For more information on the differences between using images and containers, and how to attach additional terminals to a running container, we refer to the docker documentation (e.g. \url{https://docs.docker.com/engine/getstarted/step_two/}).
+
+\subsection{Virtual Machine}
+
+\subsubsection{Installing VM software and setting up the virtual machine}
+
+We are providing an experimental virtual image for a fully configured version of \aspect{}. For this, you will need to install VirtualBox(\url{http://www.virtualbox.org/}) on your machine, and then import a virtual machine image that can be downloaded from \url{http://www.math.clemson.edu/~heister/dealvm/}. Note, however, that the machine image is several gigabytes in size and downloading will take a while. After downloading and installing the virtual image it is convenient to set up a shared folder between your host system and the virtual machine to exchange model files and outputs.
```

Comment:
I'm not fond of the "We are" way of writing things (who is "we" anyway). I would say "The \aspect{} project is providing..."